### PR TITLE
FIX: When encrypt is enabled, secure message by default

### PR DESCRIPTION
**Pull request by lis2**
When public/private keys are generated, the private message should be secured by default.

Also, I found a bug. I needed to rename `targetUsername` to `targetRecipients` due to that commit https://github.com/discourse/discourse/commit/eef21625c6b9dd589573bc1771c7075a12e0abc9

Now, when a user without generated keys are added to PM, the secure message becomes unsecured

iLg6YMXgEV mp4

### CODE REVIEW

File: assets/javascripts/discourse/initializers/hook-composer.js.es6 [hunk 1]
```diff
@@ -47,17 +47,17 @@ export default {
 
       @on("init")
       initEncrypt() {
-        this.updateEncryptProperties();
+        this.updateEncryptProperties(true);
```
Comment (SamSaffron) [inline assets/javascripts/discourse/initializers/hook-composer.js.es6]: true is a bit surprising here... `this.updateEncryptProperties({newMessage: true})` though more verbose is far more easy to follow.

File: assets/javascripts/discourse/initializers/hook-composer.js.es6 [hunk 2]
```diff
@@ -66,7 +66,7 @@ export default {
           return;
         }
 
-        const usernames = this.targetUsernames.split(",");
+        const usernames = this.targetRecipients.split(",");
```
Comment (SamSaffron) [inline assets/javascripts/discourse/initializers/hook-composer.js.es6]: nice fix!

**Comment by SamSaffron** [comment]
cool, lets try this out, feel free to merge!

**Comment by udan11** [comment]
What was the bug you are talking about? I was under the impression that there is a ['shim'](https://github.com/discourse/discourse/blob/d7d4612b2df2b4f87dfe64aafdcda8b8e8142c18/app/assets/javascripts/discourse/models/composer.js.es6#L349-L355) to keep it working.

**Comment by lis2** [comment]
@udan11 when you add a person to encrypted message who don't have keys it is erroring instead of having nice message next to the name. 

secure_bug

**Pull request merged**